build.settings.ps1

```diff
@@ -54,6 +54,29 @@ Properties {
         'My Custom Host with scriptanalyzer support'
     )
 
+    # ------------------- Testing properties ----------------------------------
+
+    # To control the build with code coverage, the $CodeCoverageStop variable can be used.
+    # The supported values for this variable are $true, $false and $null. $true enables
+    # code coverage, with the $CodeCoveragePercentage variable to reach (info below).
+    # $false disables code coverage entirely, while $null enables code coverage, but will
+    # only report on coverage status and not stop the build.
+    [System.Diagnostics.CodeAnalysis.SuppressMessage('PSUseDeclaredVarsMoreThanAssigments', '')]
+    $CodeCoverageStop = $null
```

Comment:
I just left it that way from testing. Code coverage is not something that I've ever used a lot, so we should probably start it with `$false`. It seems to more than double the testing time too, so that strengthens that 'default off' approach.